o2a/converter/renderers.py

```diff
@@ -144,6 +143,8 @@ def _remove_unused_imports(output_file_name):
 
     @staticmethod
     def _format_with_black(output_file_name):
+        import black
```

Comment:
What is the reason behind this change?

Reply:
Importing black at top level causes INFO messages being printed when you run o2a --help (bug in py2to3 library). I can separate it out if you think it's worth it :)

Reply:
And BTW. The fix does not work well with tests :(